strings/base_string_input.h

```diff
@@ -38,35 +35,23 @@ WINRT_EXPORT namespace winrt::param
         }
 
     private:
-        int32_t create_string_reference(wchar_t const* const data, size_t size) noexcept
+        void create_string_reference(wchar_t const* const data, size_t size) noexcept
         {
             auto size32 = static_cast<uint32_t>(size);
```

Comment:
This could also overflow, technically, but it's only called from some other functions. Perhaps an assert is good enough here?

Reply:
Assert it is. 👍